### ISSUE TITLE
Help & Info Page Fails To Load

### ISSUE THREAD

**Issue by DanGarion** [pymedusa/Medusa#6352]
**To Reproduce**
1. Go to Help & Info page
2. 
3. 
4.  Displays only this info.
Loading…
If this is taking too long,
click here to show the page.
**{{ $route.meta.header }}**
35 Shows (35 Active) | 1780 +6 Snatched  / 3382 Episodes Downloaded (52.6%) | Daily Search: 0:18:22 | Backlog Search: 8:58:09
Memory used: 275.30 MB | Load time: 0.0335s / Mako: 0.0309s | Branch: master | Now: 03/14/19, 02:20 PM

**Expected behavior**
Help & Info page should display.

**Screenshots**
![image](https://user-images.githubusercontent.com/1666375/54392280-77acd480-4664-11e9-9cdd-e2d0fd6643a5.png)

**Medusa (please complete the following information):**
 - OS: Debian /// Docker container
 - Branch: Master
 - Commit: 918cfe7 ? I think I can't confirm as I can't view this page.

**Additional context**
Issue did not occur till the most recent update.

**Comment by p0psicles** [comment]
Browser plus version? Also any errors in in your browsers console? Press f12

**Comment by DanGarion** [comment]
> Browser plus version? Also any errors in in your browsers console? Press f12
Chrome Version 72.0.3626.121 (Official Build) (64-bit) and Version 73.0.3683.75 (Official Build) (64-bit) 
I did a bit more testing afterward and I forced a full page refresh using ALT-F5 and that appears to have fixed the issue...

I checked in Firefox 66.0b14 (32-bit) and it doesn't happen.
I also checked in Chrome Version 75.0.3733.0 (Official Build) canary (64-bit) and it doesn't happen. SO it appears to just be something with the current Chrome builds not beta...

----- So long story short, this ticket is closed. :)
